ex6_ch2.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: ask user for input: age
+current_age = int(input("What is your age? "))
+
+#step2: ask user for input: ideal retirement age
+ideal_retirement_age = int(input("What age do you want to retire? "))
+
+#step3: write calculation to determine the number of years until retirement
+years_to_retirement = ideal_retirement_age - current_age
+
+#step4: write calculation to determine the year you can retire
+retirement_year = 2018 + years_to_retirement
```

Comment:
A possible refinement is making `2018` a constant:

```
CURRENT_YEAR = 2018
```

And then you can use it here and on line 25. This eliminates duplication so when the year changes, you can change it in one place instead of 2.

Reply:
Great point!